src/main/java/org/ambraproject/rhino/service/impl/ArticleCrudServiceImpl.java

```diff
@@ -665,8 +665,10 @@ public void updatePreprintDoi(ArticleIngestionIdentifier articleId, String prepr
 
         if (sortOrder == SortOrder.OLDEST) {
           criteria.addOrder(Order.asc("created" /* propertyName */));
+          criteria.addOrder(Order.asc("articleId"));
         } else {
           criteria.addOrder(Order.desc("created" /* propertyName */));
+          criteria.addOrder(Order.asc("articleId"));
         }
```

Comment:
Why not just one line after the if/else blocks?

Reply:
Good point.

Reply:
Fixed.